monorepo/website/src/pages/docs/how-to/use-tools-and-add-new-ones.mdx

```diff
@@ -0,0 +1,79 @@
+---
+layout: ../../../layouts/DocLayout.astro
+title: 'Use external tools and suggest new ones'
+order: 105
+---
+
+
+## Using the Tools menu
+
+The sequence search page contains a **Tools** dropdown that lists external
+applications that can be launched with the sequences in your search query or any that you select. Select some sequences (or just make a search), open the
+dropdown and choose a tool. You will be taken to the tool in question which will be supplied with URLs so that it can seamlessly retrieve the sequences you are looking for from Pathoplexus.
+
+Initial tools allow you to:
+- perform QC on sequences with Nextclade and see them on a tree (for some organisms)
+- visualise the geographic distribution of your sequences
+
+## Configuring a new tool to be added to Pathoplexus
+
+We are interested in adding new useful external tools to this menu. If you are a tool developer interested in having your tool featured here, please read on.
+
+### Requirements
+
+For a tool to be integrated into the Pathoplexus tools menu, it must be able to:
+
+- Accept file URLs as parameters in its web address
+- Automatically fetch and process the files from these URLs and do something useful with them
+
+When users select your tool from the Tools menu, Pathoplexus will generate download URLs, configured in a way that you specify, then redirect users to your tool with these URLs embedded in the web address. Your tool needs to parse these URLs from its incoming request and retrieve the files directly from Pathoplexus.
+
```

Comment:
```suggestion
Tools are available on a per-pathogen basis; if a tool will only work with some of the pathogens on Pathoplexus, you can add it to just those. If it will work on all pathogens, please add it to every pathogen in your PR.

```

Reply:
I updated because it seemed a bit unexpected as we hadn't mentioned a "PR" yet - can iterate further!